server/datastore/mysql/hosts.go

```diff
@@ -2328,17 +2328,6 @@ func (ds *Datastore) LoadHostByOrbitNodeKey(ctx context.Context, nodeKey string)
 		host := hostWithMDM.Host
 		// leave MDMInfo nil unless it has mdm information
 		if hostWithMDM.HostID != nil {
```

Comment:
we should cleanup `hostWithMDM` too, at least remove the unused fields for now and come back later for `EncryptionKeyAvailable`

Reply:
Done